tests/ai_yolo/test_yolo_model.py

```
@@ -0,0 +1,226 @@
+        results = yolo_glaucoma_model.predict(
+            '../../tests/ai_yolo/1.png', save=False, conf=0.25, verbose=False
+        )
+        xyxy = results[0].boxes.xyxy.cpu().numpy()
+        assert len(xyxy) > 0, "No detections to validate"
+
+        x1, y1, x2, y2 = xyxy[0]
+        width = x2 - x1
+        height = y2 - y1
+        assert width > 0 and height > 0, (
+            f"Detection box has non-positive dimensions: w={width:.1f}, h={height:.1f}"
+        )
```

Ruff (0.15.15)

[warning] Assertion should be broken down into multiple parts

(PT018)